_includes/header.html

```diff
@@ -72,12 +78,13 @@
         <div class="module-group right">
           <div class="module left">
             <ul class="menu">
-              <li><a href="#welcome">Welcome</a></li>
-              <li><a href="#mentors">Mentors</a></li>
-              <li><a href="#students">Students</a></li>
-              <li><a href="#contributors">Contributors</a></li>
+              <li><a href="index.html#welcome">Welcome</a></li>
```

Comment:
No need to change these. We need scrolling behaviour. Undo for other nav links also.